enterprise/frontend/src/embedding-sdk/lib/default-height.ts

```diff
@@ -0,0 +1,18 @@
+import type { CardDisplayType } from "metabase-types/api";
+
+/**
+ * Default height for each visualization type.
+ *
+ * The values are derived from the visualization's default size in the dashboard.
```

Comment:
@WiNloSt Didn't knew that there is a defined card size in code, nice! As there is no fixed dimension in the Figma file, this is just a rough estimate based on measuring the visualization itself from top to bottom, excluding the card itself and other elements.

Reply:
could it make sense to use this value instead of hard-coding all the values? e.g. we could say something like, one block equals 50px so 4 by 3 blocks is 200px by 150px = 150px tall.

You can import it like this
https://github.com/metabase/metabase/blob/f745c53d4d929e40576983991357d7b1887eda16/frontend/src/metabase/visualizations/shared/utils/sizes.ts#L3

Reply:
@WiNloSt Let me try this out! I think that should do the trick, the sizing might deviate a bit from what we see in the dashboard just because we render the raw visualization - but should be close enough.